blockchain/genesis/genesis.go

```diff
@@ -495,6 +500,11 @@ func (g *Blockchain) IsLordHowe(height uint64) bool {
 	return g.isPost(g.LordHoweBlockHeight, height)
 }
 
+// IsToBeEnabled checks whether height is equal to or larger than toBeEnabled height
```

Comment:
Could we add more details about what toBeEndabled feature do?

Reply:
explained in `genesis.go`
new features will use `toBeEndabledHeight` so they are disabled by default, before release, replace it with actual height name